projects/libtorrent/build.sh

```diff
@@ -15,14 +15,26 @@
 #
 ################################################################################
 
-#create zip files with initial corpus, taken from version control.
-for f in $(ls fuzzers/initial_corpus/) ;do
-  zip -j -r $OUT/fuzzer_${f}_seed_corpus.zip fuzzers/initial_corpus/$f
-done
+export PATH=${PATH}:${PWD}/../boost_1_69_0/tools/build/src/engine/bin.linuxx86_64
+export BOOST_ROOT=${PWD}/../boost_1_69_0
+export BOOST_BUILD_PATH=${PWD}/../boost_1_69_0/tools/build
+
+(cd ${PWD}/../boost_1_69_0 && ./bootstrap.sh)
+
+if [[ $LIB_FUZZING_ENGINE == -fsanitize=fuzzer ]]; then CXXFLAGS="${CXXFLAGS} -fsanitize=fuzzer"; fi
```

Comment:
Please don't try to add -fsanitize=fuzzer to CXXFLAGS, -fsanitize=fuzzer-no-link should take care of this.